### PR TITLE
Backport featureimage/deb10 progress to master branch

### PR DESCRIPTION
**Pull request by jimklimov**
Bookmarking the featureimage codebase state that passed "quick boot+networking" tests well in oldOVA+newOVA+LXC, before I break it again :)

Rebased over current master for a cleaner merge.

### CODE REVIEW

File: database/mysql/0019_discovery_monitoring.sql [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+ *
+ * This software is not authorized to be used, duplicated or disclosed to
+ * anyone without the prior written permission of Eaton.
+ * Limitations, restrictions and exclusions of the Eaton applicable standard
+ * terms and conditions, such as its EPL and EULA, apply.
+ *
+ */
+
+/*
+ * @file    0019_discovery_monitoring_v2.sql
```
Comment (boricj) [inline database/mysql/0019_discovery_monitoring.sql:16]: Is the new NUT configurator expected to be merged right now? I was under the impression it had to be modified to switch from an asset-centric to an endpoint-centric approach.

File: database/mysql/0019_discovery_monitoring.sql [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+ * @brief   User preferences builtin data.
```
Comment (boricj) [inline database/mysql/0019_discovery_monitoring.sql:17]: Wrong brief.

File: database/mysql/0019_discovery_monitoring.sql (hunk 1 above)
Comment (jimklimov) [reply database/mysql/0019_discovery_monitoring.sql:16]: Thanks for the nudge, maybe the file leaked from some other work or experiment, not sure it was destined for deb10 specific work indeed. At least does not seem to fit to the subject of https://github.com/42ity/fty-core/commit/0e3e835c79b697bd850973c35b7287aa509b538c (the `/var/run => /run` changes) so might be just something `git add`ed in the workspace and forgotten :\

Comment (jimklimov) [reply database/mysql/0019_discovery_monitoring.sql]: Rebased to drop it from the PR.

**Pull request merged**